contracts/starknet/space.cairo

```diff
@@ -113,6 +113,38 @@ func update_controller{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_c
     return ()
 end
 
+@external
+func add_executors{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr : felt}(
```

Comment:
can we get away with just `add_executors` or just `register_executors` ? seems kind of redundant to have both

Reply:
lmao I'm stupid! Ofc we can :) 843ad89e8578e0277571fd40cbcb0d15554a4c63

Reply:
Oh actually the reason why there was an `add_executors` and a `register_executors` was because `add_executors` is `only_controller()` whereas `register` is not. Since `_controller` is an argument passed in to the `constructor` (and not `get_caller_address`), `add_executors` fails when called in the `constructor`... I think we should keep `register_executors`, but we could decide to go the other way and simply set the `_controller` to `get_caller_address` in the constructor

Reply:
Decided to fix it by having `add_executors` simply be a wrapper with a call to `only_owner()` before `register_executors`. I think naming should be changed though, and maybe use `_` as we sometimes do in solidity :)

23440683e044789d60d27dfa6f5071436ebe4c97